wildfly-mixin/dashboards/wildfly-overview.libsonnet

```
@@ -0,0 +1,711 @@
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local expiredSessionsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'increase(wildfly_undertow_expired_sessions_total{deployment="$deployment", job="$job", instance="$instance"}[$__rate_interval])',
+      datasource=promDatasource,
+      legendFormat='{{deployment}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Expired Sessions',
+  description: 'Number of sessions that have expired for a deployment over time',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      decimals: 0,
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local rejectedSessionsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'increase(wildfly_undertow_rejected_sessions_total{deployment="$deployment", job="$job", instance="$instance"}[$__rate_interval])',
+      datasource=promDatasource,
+      legendFormat='{{deployment}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Rejected Sessions',
```

```suggestion
  title: 'Rejected sessions',
```

All panels are sentence case.